engine/src/main/java/org/pentaho/di/trans/StepWithMappingMeta.java

```diff
@@ -248,6 +249,15 @@ public static void activateParams( VariableSpace childVariableSpace, NamedParams
           // this is explicitly checked for up front
         }
       } else {
+        try {
+          childNamedParams.addParameterDefinition( key, "", "" );
+          childNamedParams.setParameterValue( key, value );
+        } catch ( DuplicateParamException e ) {
+          e.printStackTrace();
```

Comment:
No to e.printStackTrace() - these should use the standard logging mechanism, not print a stack trace